packages/mobile/src/account/StoreWipeRecoveryScreen.tsx

```diff
@@ -50,6 +52,7 @@ function StoreWipeRecoveryScreen({ route }: Props) {
 
 StoreWipeRecoveryScreen.navOptions = {
   ...emptyHeader,
+  headerLeft: () => null,
```

Comment:
nit: could `noHeader` or `noHeaderGestureDisabled` work better here?

Reply:
I could have sworn I tried `noHeader` and it didn't work but I tried it again and it worked. I must have done something wrong the first time :)